packages/cli/src/credentials/credentials.service.ee.ts

```diff
@@ -87,7 +87,7 @@ export class EnterpriseCredentialsService {
 		if (credential) {
 			// Decrypt the data if we found the credential with the `credential:update`
 			// scope.
-			decryptedData = this.credentialsService.decrypt(credential, true);
+			decryptedData = this.credentialsService.decrypt(credential, false);
```

Comment:
The 2nd parameter defaults to `false` so you can leave it off.

Reply:
Fixed